templates/app/scripts/offline-manager.js

```diff
@@ -27,6 +27,10 @@ function updateFound() {
 if ('serviceWorker' in navigator) {
   navigator.serviceWorker.register('offline-worker.js').then(function(registration) {
     console.log('offline worker registered');
+    // Check if there's an updated version of service-worker
```

Comment:
Please, remove the comment and the check (as all the browsers supporting SW have `update()` implemented) and put the `update()` invocation after registering the event.